documentation/docs/pipelines/cloud-sdk/configuration.md

```diff
@@ -682,30 +549,6 @@ checkJMeter:
   unstableThreshold: 70
 ```
 
-### detectExecuteScan
-
-To activate a Synopsys Detect Scan, the step `detectExecuteScan` has to be configured in the steps section.
-Please visit the project "Piper" [documentation](https://sap.github.io/jenkins-library/steps/detectExecuteScan/) for the configuration options of this step.
-
-### fortifyExecuteScan
-
-The configuration of the step fortifyExecuteScan is explained in the project "Piper" [documentation](https://sap.github.io/jenkins-library/steps/fortifyExecuteScan/).
-
-### mtaBuild
-
-| Property | Mandatory | Default Value | Description |
-| --- | --- | --- | --- |
-| `dockerImage` | | `ppiper/mta-archive-builder` | Docker image including Multi-target Application Archive Builder. The default image is `ppiper/mta-archive-builder` for mtaBuildTool `classic`. And it takes `devxci/mbtci:1.0.4` as a default image for mtaBuildTool `cloudMbt`. |
-| `mtaBuildTool` | | `cloudMbt` | Choose which tool is used to build your mta project. The default option is `cloudMbt` which is not backwards compatible with the `classic` tool. For more information on migrating from `classic` to `cloudMbt`, please refer to [the Cloud MTA Build Tool docs on the topic of "migration"](https://sap.github.io/cloud-mta-build-tool/migration/). |
-
-All configuration parameters as stated in [jenkins-library documentation](https://sap.github.io/jenkins-library/steps/mtaBuild/) are available.
-
-### tmsUpload
-
-The `tmsUpload` step can be used to upload your app during the production deployment stage using SAP Cloud Platform Transport Management.
-If the step is configured, SAP Cloud Platform Transport Management upload will be executed in the production deployment stage.
-Further information can be found in the [project "Piper" documentation](https://sap.github.io/jenkins-library/steps/tmsUpload/).
-
 ### debugReportArchive
```

Comment:
This one is also a project "Piper" step, actually.  At the moment, only our pipeline uses it in the post phase.

Reply:
yes, but it is not yet documented. I'll merge and we can create dedicated step config later, if desired. I'm not sure if we need/should have that in the first place.

Reply:
> yes, but it is not yet documented. I'll merge and we can create dedicated step config later, if desired. 

OK

> I'm not sure if we need/should have that in the first place.

True enough.